uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/SicknessRequestTest.php

```diff
@@ -207,4 +219,60 @@ public function testValidateParamsCallsLeaveRequestValidateParams() {
       'required_documents' => $requiredDocuments
     ]);
   }
+
+  public function testCreateAndUpdateSicknessRequest() {
```

Comment:
Is this test really necessary? We don't do any special handling for reason or required_documents, so there's nothing that needs to be tested. I understand you added this because of the possible bug reported by Dhaval, but since there isn't really a bug (at least, not one we could reproduce), then no test is needed I would say.